### DISCUSSION TITLE
Extend Prisma Postgres to generate types for Postgres functions/stored procedures

### DISCUSSION THREAD

**Discussion by dangdennis** [prisma/prisma#7532] [Q&A]
Is it possible to extend the Prisma Postgres type generation to include plpgsql functions?

**Comment by ryands17** [comment]
@dangdennis 👋 

Currently this isn't possible, but I would suggest creating a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
